insights/attachments/attachment_yara_rule_match.yml

```diff
@@ -0,0 +1,11 @@
+name: "Yara rule match"
+type: "query"
+source: |
+  map(filter(attachments,
+           .file_type in $file_extensions_common_archives
+           or .file_type in $file_extensions_executables
+           or .file_type == "unknown"
+    ),
+    map(file.explode(.), distinct(map(.scan.yara.matches, .name), .))
+  )
+severity: "medium"
```

Comment:
this is interesting... the severity of the yara sig could be anything. but idk what else we would put here

Reply:
I figure if you have yara installed, you probably want to know about any matches. Informational didn't seem strong enough, and critical seemed like overkill.